site/it/community/contribute/code.md

```diff
@@ -0,0 +1,136 @@
+# Contributo al codice TensorFlow
+
+Sia che tu voglia aggiungere una funzione perdita, migliorare la copertura di un test, oppure scrivere una
```

Comment:
I don't think loss is translated

Reply:
Also check https://accademiadellacrusca.it/it/consulenza/genere-dei-forestierismi/1229 about cases like this one. Is it more common `l'RFC/un RFC` or `la RFC/una RFC`?

Reply:
> I don't think loss is translated

loss function in Italian is "funzione obiettivo": https://it.wikipedia.org/wiki/Funzione_obiettivo

Reply:
> Also check https://accademiadellacrusca.it/it/consulenza/genere-dei-forestierismi/1229 about cases like this one. Is it more common `l'RFC/un RFC` or `la RFC/una RFC`?

Checked. I think RFC is properly used.

Reply:
`Funzione obiettivo` It Is used but It Is not so dominant. I don't have stats but as we will find this quite frequently It Is better to collect other reviewers feedbacks here and choice If we want to put in the untrslated list or not.